.github/workflows/sauce-ie11.yaml

```diff
@@ -42,5 +42,7 @@ jobs:
           SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
           SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
           BROWSERS: preset:sauce-ie11
-        # Retry once automatically.
-        run: npm run test || npm run test
+        run: |
+          export SAUCE_BUILD_ID=lit-tests__${GITHUB_EVENT_NAME}__${GITHUB_SHA}
+          export SAUCE_TUNNEL_ID=${SAUCE_BUILD_ID}_tunnel
```

Comment:
Is there a page we could link to that describes `SAUCE_TUNNEL_ID`? Is that read by web-test-runner? (I couldn't find it from a quick google).

Reply:
Ooops! Forgot to track a file change for wtr, will update shortly

Reply:
Appears we had a pretty successful run regardless